playbooks/deploy.yaml

```
@@ -51,9 +49,9 @@
       - name: candlepin
         password: "{{ candlepin_db_password }}"
       - name: foreman
```

We now have a variable for this:
```suggestion
      - name: "{{ foreman_database_user }}"
```